html/semantics/forms/the-input-element/range.html

```diff
@@ -222,7 +222,7 @@ <h1>Input Range</h1>
       test(
         function() {
           var e = document.getElementById('should_skip_whitespace');
-          assert_equals(e.value, "50")
+          assert_equals(e.value, "49")
```

Comment:
I haven't tried Chrome / Firefox but this is failing in Safari at least. Curious if this is a bug in WebKit or if other engines are aligned.

Reply:
This is failing in all 3 browsers actually so the browsers are aligned in not skipping the leading space here.

Seems this test change doesn't align it with major browser engines.

Reply:
Yep, chrome also changes `" 49"` to `"50"`.
I guess that the desired behavior seems sane, but I'm also open to deleting the test since the behavior is already interoperable if that's what people want to do.
I read into this more, and apparently the spec says that we should be passing this test: https://github.com/servo/servo/pull/19761#discussion_r161468512

Reply:
The reason is this, I think:

> The [value sanitization algorithm](https://html.spec.whatwg.org/#value-sanitization-algorithm) is as follows: If the [value](https://html.spec.whatwg.org/#concept-fe-value) of the element is not a [valid floating-point number](https://html.spec.whatwg.org/#valid-floating-point-number), then set it to the [best representation, as a floating-point number](https://html.spec.whatwg.org/#best-representation-of-the-number-as-a-floating-point-number), of the [default value](https://html.spec.whatwg.org/#concept-input-value-default-range).

The leading space makes it invalid and therefore the default value ends up being used. So I think what we want is to update the test description only. Or is there another way to read the specification here?

Reply:
Ok, let's go back to 50 and update the test description then. Thank you both.

Reply:
Ok, I changed the value back to " 123" and the expected value to 50, and updated the test description. Does it look good?